core/src/main/scala/org/apache/spark/eventhubs/client/CachedEventHubsReceiver.scala

```diff
@@ -83,6 +85,7 @@ private[client] class CachedEventHubsReceiver private (ehConf: EventHubsConf,
       logWarning(
         s"$requestSeqNo did not match ${event.getSystemProperties.getSequenceNumber}." +
           s"Recreating receiver for $nAndP")
+      _receiver.closeSync()
```

Comment:
>_receiver.closeSync() [](start = 6, length = 21)

if close fails - we still want to go ahead and create the new Receiver..
So, lets absorb any non-fatal errors - and log'em.